tutorial/practicals/LFRic/Makefile

```diff
@@ -0,0 +1,45 @@
+# -----------------------------------------------------------------------------
+# BSD 3-Clause License
+#
+# Copyright (c) 2020, Science and Technology Facilities Council.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+# FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+# COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+# ------------------------------------------------------------------------------
+# Authors: A. R. Porter and R. W. Ford, STFC Daresbury Lab
+
+# Top-level Makefile to test all of the LFRic hands-on tutorials.
+
+TUTORIAL_DIRS = $(wildcard */?_*)
+
+.PHONY: test ${TUTORIAL_DIRS}
+
+test: ${TUTORIAL_DIRS}
+
+${TUTORIAL_DIRS}:
```

Comment:
OK, that required a bit of work: I don't have a 'normal' makefile, I have one 'Makefile.inc', which is included from transformation-specific Makefiles like Makefile.extract, Makefile.readonly etc. The latter have just three lines (and one include) in them.

This simplifies the job for the user, since the makefile they have to modify is really really small. And I prevent them from accidentally starting the wrong compilation with unexpected psyclone options by not providing a Makefile. But I like the testing of course, and have added a dummy Makefile that prints instructions when invoked without a target, but 'make test' will invoke Iva's test target. I will change this as part of #991 (where I have added this as a comment).